nn-git-log/README.md

```diff
@@ -0,0 +1,113 @@
+# Git Log
+
+Now that you know how to do the basics in git and started working on the repo, a few commits in and you might ask yourslef, how do we visualize these commits and changes we have made in the repository?  
+
+That is where this chapter, and the subsequent, `Git Diffs` come in. These commands give you a way to visualize the commits and changes made in your git respository.  
+
+To explain simply, the command `git log` and its corresponding optional flags lets you look at history of commits and how the link up in your respository. At the end of the chapter there will be links to additional resources, as well as more visual GUI tools you can use to look at the log of the respository, but I still suggest reading and learning this chapter.  
+
+This is because these tools still do rely on the git commands to generate their graphs, and understanding this command kind of gives you the 'feel' for it in general. Lastly, as this command comes with git, using the command does not require any additional setup other than having git.
+
+Note:
+This chapter serves to show you and explain the common use cases for the command. For a more detailed explaination, look at the addtional resources or the documentation of the command.
+
+## Project set up so-far
+
+Let's say the project you are working on is a full-stack web application, with a frontend and backend.  
+And you have decided to split the work to frontend and backend work, with respective features branches being worked on.
+
+It may look something like this:  
+```
+Github
```

Comment:
```
Github branches:
master, start-server (This will be remote branches for you)

Teammate A's branches (On your local machine):
master, origin/master, backend/start-server, origin/backend/start-server, frontend/landing

Teammate B's branches:
backend/start-server (His local branch connected to github)
```

Tried to be more explicit, but I'm not sure if this helps

Reply:
Yeap, these changes work :+1: Much clearer as to what we are seeing